config.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"io/ioutil"
+	"encoding/json"
+)
+
+type Config struct {
+	Sources map[string]string `json:"sources"`
+}
+
+func ReadConfig(data []byte) (*Config, error) {
+	var config Config
+	err := json.Unmarshal(data, &config)
```

Comment:
I wasn't going to go there because it requires an external dependency, but it occurs to me that we'll need YAML handling anyway, so... yeah, OK. :)